### PR TITLE
When adding a course in admin non numeric id results in 500 error

### LINKED ISSUE #459: When adding a course in the course admin adding a non-numeric id results in 500 error
If you're in the admin interface and add a course using a non-numeric id (like using text, "test") it says

**Server Error (500)**

This error should be handled better rather than throwing this error

### PR DESCRIPTION
**Pull request by jonespm**
Fixes #459

### CODE REVIEW

File: dashboard/admin.py [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
             exclude += (view,)
 
+class CourseForm(forms.ModelForm):
+    class Meta:
+        model = Course
+        exclude = ()
+
+    def clean(self):
+        canvas_id = self.cleaned_data.get('canvas_id')
+        if not canvas_id.isdigit():
```
Comment (andrew-gardener) [inline dashboard/admin.py]: `if not canvas_id or not canvas_id.isdigit():` (leaving blank causes error)

File: dashboard/admin.py [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+            raise forms.ValidationError("Canvas ID must be an integer value")
```
Comment (pushyamig) [inline dashboard/admin.py]: I think we should log the course_id so the message should say 
```suggestion
            raise forms.ValidationError(f"Course ID {canvas_id} must be an integer value")
```

File: dashboard/admin.py (hunk 1 above)
Comment (jonespm) [reply dashboard/admin.py]: Thanks, updated!

File: dashboard/admin.py (hunk 2 above)
Comment (jonespm) [reply dashboard/admin.py]: Thanks! Updated.

**Pull request merged**